tests/headers/basicauth_test.go

```diff
@@ -180,3 +163,65 @@ func TestBasicAuth(t *testing.T) {
 		})
 	}
 }
+
+func TestBasicAuthOrdering(t *testing.T) {
+	// The documentation of http.Request.BasicAuth() doesn't clearly specify
+	// that only the first Authorization header is used and that the other ones
```

Comment:
Simplify to "the other ones ignored" so it's easier to read :)

Reply:
Do you mean this change?
`and that the other ones are ignored.` -> `and that the other ones ignored.`
That feels less grammatically correct. 
I agree that we can remove the `that` though.